manual/src/installation.md

```diff
@@ -97,3 +97,5 @@ Users of older MacOS versions (e.g. 10.11 El Capitan) should install using Homeb
 Behind the scenes, delta uses [`less`](https://www.greenwoodsoftware.com/less/) for paging.
 It's important to have a reasonably recent version of less installed.
 On MacOS, install `less` from Homebrew. For Windows, see [Using Delta on Windows](./tips-and-tricks/using-delta-on-windows.md).
+
+Delta also relies on [`bat`](https://github.com/sharkdp/bat) for [custom themes and syntaxes](./supported-languages-and-themes.md). Ideally, you should install the same version of bat as specified in the [`Cargo.toml`](https://github.com/dandavison/delta/blob/main/Cargo.toml) for the delta release you're using. There are [known problems](https://github.com/dandavison/delta/issues/1712) with v0.18.3 and prior versions of bat.
```

Comment:
For a long time there's been quite common confusion that delta uses the bat executable somehow. How about this wording:

```suggestion
If you use [`bat`](https://github.com/sharkdp/bat) and are running `bat cache --build` to install custom themes or language syntaxes then you should install the same version of bat as specified in the [`Cargo.toml`](https://github.com/dandavison/delta/blob/main/Cargo.toml) for the delta release you're using, otherwise delta will crash with a [memory error](https://github.com/dandavison/delta/issues/1712). The current version of delta does not work with bat v0.18.3 or prior versions.
```

Reply:
That's good. I considered adding "relies on bat _as a library_" but for some reason thought twice about that.

I get that anything but "the current version" is ancient history for a project that hasn't reached 1.0 yet, but is it significant or worth mentioning that the same problem occurs with every version of delta back to 0.16.5 (the last one to include a musl binary in the release assets that I could easily download and run)?

Reply:
I think what you have is good. Thanks for this PR!